infra/modules/database/role-manager.tf

```diff
@@ -49,10 +49,11 @@ resource "aws_lambda_function" "role_manager" {
 }
 
 # Installs python packages needed by the role manager lambda function before
-# creating the zip archive. Reinstalls whenever requirements.txt changes
+# creating the zip archive. 
+# Runs pip install on every apply.
```

Comment:
Can we add an explanation for why we run on every apply? Maybe something about making sure that the lambda archive file contains the required dependencies even when the terraform plan/apply is run by a developer that did not originally create the environment.

```suggestion
# Runs pip install on every apply so that the role manager archive file that
# is generated locally is guaranteed to have the required dependencies even
# when terraform is run by a developer that did not originally create the
# environment.
```